### ISSUE TITLE
Feature Request: Icon and title position as a normal tabbar

### ISSUE THREAD

**Issue by lamlv2305** [torgeadelin/react-native-animated-nav-tab-bar#45]
<img width="739" alt="Screen Shot 2020-10-06 at 01 17 36" src="https://user-images.githubusercontent.com/7299214/95116764-bab42880-0771-11eb-9ad8-f762a479189d.png">

Can I place the title and icon as the above image?

**Comment by torgeadelin** [comment]
Hi, at the moment no. We can add this in the future but I'm not sure how the animation will look with that layout.

**Comment by lamlv2305** [comment]
@torgeadelin Thanks for your response.

I'm trying to custom the layout without animation, just only float tab bar, icon and title.

For extra animation, I will use another approach like: https://github.com/software-mansion/react-native-reanimated

**Comment by amarjitsingh65** [comment]
Hi, I 
<img width="318" alt="floating-bottom-bar" src="https://user-images.githubusercontent.com/34943963/95499173-bd717200-0959-11eb-9a1b-c2233072211a.png">
am not able to create a floating tab with less rounding corners. how you rounder the corner with less radius as in the above screen. I have a default radius which is too large.
@Zoka. how you created above tab bar with shadow and less corner, I tried different properties but doesn't work

**Comment by torgeadelin** [comment]
@amarjitsingh65 to change the corner-radius for the tabbar wrapper you have to use the tabBarOptions props and give the tabStyle a specific radius.
```
tabBarOptions={{
    tabStyle: {
        borderRadius: 10,
    },
}}
``` 

For shadow, please check the docs.

**Comment by torgeadelin** [comment]
Fixed #48. Please upgrade to 3.0.5 and use the new `tabButtonLayout` to change the layout of the button. For you @zoka2305 you'll need `vertical`, default is `horizontal`.

PS. you won't be able to change the animation as far as I know.
